### DISCUSSION TITLE
No covers / list view for albums

### DISCUSSION THREAD

**Discussion by iconoclasthero** [jcorporation/myMPD#1247] [Ideas]
I hate album art.  Can't there be a list option, e.g. nautilus:
![Screenshot from 2024-03-27 03-16-23](https://github.com/jcorporation/myMPD/assets/535962/2cc99334-4bea-4c35-ab69-eb6d5084012a)

I just really, really hate having all those pictures there wasting space i could be using to find out what the albums are by reading characters like the ones you're reading right now.

**Comment by jcorporation** [comment]
That is already in planing, v15.0.0 should have this option: https://github.com/jcorporation/myMPD/issues/1051
